cmd/testutils/dbedit/dbedit.go

```diff
@@ -115,8 +115,6 @@ func resetRs(db *db.Database, txn *badger.Txn, height uint32) {
 
 func ovsFn(ovs *objs.OwnValidatingState) *objs.OwnValidatingState {
 	ovsin := &objs.OwnValidatingState{
-		VAddr:                ovs.VAddr,
```

Comment:
What is the reasoning behind removing these?

Reply:
Hunter removed this. Apparently this was not necessary there.